### PR TITLE
Allow user to provide custom buttons to the form

### PR DESCRIPTION
**Pull request by cedricmessiant**
Refs #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,6 +231,20 @@ render((
...
 
+## Custom buttons
+
+You can provide custom buttons to your form via the `Form` component's `children`. A default submit button will be rendered if you don't provide children to the `Form` component.
+
+```jsx
+render(
+  <Form schema={schema}>
+    <div>
+      <button type="submit">Submit</button>
```
Comment (n1k0) [inline README.md]: We should add an important note that there should be either a button or input with `type=submit` in order to trigger the form submission.

File: test/index_test.js [hunk 2]
```diff
@@ -43,6 +43,20 @@ describe("Form", () => {
...
+      const comp = renderIntoDocument(
+        <Form {...props}>
+          <button type="submit">Submit</button>
+          <button type="submit">Another submit</button>
+        </Form>
+      );
+
+      const node = findDOMNode(comp);
+      expect(node.querySelectorAll("button[type=submit]"))
+      .to.have.length.of(2);
```
Comment (n1k0) [inline test/index_test.js]: nit: indentation

File: test/index_test.js [hunk 3]
```diff
@@ -43,6 +43,20 @@ describe("Form", () => {
...
+    });
   });
```
Comment (n1k0) [inline test/index_test.js]: We should now add a test ensuring than clicking on one of these custom buttons triggers form submission.

**Comment by n1k0** [comment]
Shaping great!

**Comment by cedricmessiant** [comment]
et voilà !

**Comment by n1k0** [comment]
Merveilleux! I'm landing this. Thanks for contributing :)

**Comment by n1k0** [comment]
I've just published [v0.10.0](https://github.com/mozilla-services/react-jsonschema-form/releases/tag/v0.10.0) to npm.

**Comment by cedricmessiant** [comment]
Perfect, thanks !

**Comment by mravenash** [comment]
Do we have a sandbox or plunker for this issue. Any help is appreciated.

Regards
Avenash

**Pull request merged**